webui_runner/plugs/lucet_app/download.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source ../../config.inc
+
+>&2 echo "lucet_app download"
+mkdir -p build; cd build
+if [ ! -d lucet ]; then 
+    git clone https://github.com/fastly/lucet.git lucet
+fi
```

Comment:
So if the `lucet` directory already exists, do we need to do a `pull` or `fetch` to grab the latest code?